include/vigra/linear_solve.hxx

```diff
@@ -93,6 +93,42 @@ T determinantByLUDecomposition(MultiArrayView<2, T, C1> const & a)
     return det;
 }
 
+template <class T, class C1>
+T determinantByMinors(MultiArrayView<2, T, C1> const & mat)
```

Comment:
It's better to change the return type into `typename NumericTraits<T>::Promote`. Then the function works for `UInt8` etc. as well.